src/movie/movie.ts

```diff
@@ -130,6 +130,7 @@ export class Movie {
    * @return Fulfilled when the movie is done playing, never fails
    */
   async play (options: {
+    onDraw?: () => void,
```

Comment:
Can you add `onDraw` to this method's docstring?

Reply:
Added a simple docstring 👍